### DISCUSSION TITLE
Multiple regular gaps in the interferogram after processing sentinel-1 using stackSentinel.py

### DISCUSSION THREAD

**Discussion by hopegis** [isce-framework/isce2#890] [Q&A]
Dear community,
I used stackSentinel.py to process the sentinel-1 data, and looked at the fine.int file with mdx.py and found multiple gaps (Figure). Does anyone know how these gaps are created and how to get rid of them? Thanks.
![Snipaste_2024-10-09_11-04-33](https://github.com/user-attachments/assets/f52bbbd6-d48a-430b-9943-03482f39d7c8)
![Snipaste_2024-10-09_11-07-29](https://github.com/user-attachments/assets/b0fc25e4-b832-455c-9fc4-17101b11a4ce)
Best

**Comment by piyushrpt** [comment]
Check your common valid region extraction.

**Reply by hopegis** [reply]
thanks
When I execute run_11_extract_stack_valid_region, the following message appears：


checking the number of bursts in coreg_secondarys against the one in reference
WARNING: 20220813 has different number of bursts (21) than the reference reference (27) for swath 2 --> exclude it for common region calculation
WARNING: 20220825 has different number of bursts (21) than the reference reference (27) for swath 2 --> exclude it for common region calculation
WARNING: 20220813 has different number of bursts (22) than the reference reference (27) for swath 3 --> exclude it for common region calculation
WARNING: 20220825 has different number of bursts (22) than the reference reference (27) for swath 3 --> exclude it for common region calculation
******************

**Comment by piyushrpt** [comment]
Are the pairs that you are showing corresponding to those dates?

**Reply by hopegis** [reply]
yes

**Reply by piyushrpt** [reply]
I'm not sure that the pipeline supports different number of bursts on different dates. A lot of work is done upfront usually to make sure that the entire area of interest is covered before processing is kicked off usually. 

You can skip using those dates completely - the ones that report fewer bursts.
The only other option is to actually hack the code and try to set common regions by computing it yourself and making some sort of assumptions about missing bursts.

**Reply by hopegis** [reply]
The strips involved in the workspace for the period 2022-2023 have two burst types, 21 and 26 phases, with irregular date intervals. How much does ignoring the SLC data corresponding to one of these types affect the final results of doing sbsa deformations (e.g., velocities)?
Thank you very much！
